web-portal/package.json

```diff
@@ -32,7 +32,9 @@
     "vue-clipboard2": "^0.3.0",
     "vue-moment": "^4.0.0",
     "vue-smoothscroll": "^0.2.0",
-    "vue2-editor": "^2.10.0"
+    "vue2-editor": "^2.10.0",
+    "winston": "^3.2.1",
+    "winston-daily-rotate-file": "^4.4.1"
```

Comment:
we don't actually need this file. I probably added that to mine by accident. that's something we might use in the future

Reply:
Just pushed a commit to remove `winston-daily-rotate-file`. Any objections to merging the PR?